index.html

```diff
@@ -1264,7 +1263,7 @@ <h3>Profile Conformance</h3>
 							</td>
 							<td>URL of the profile.</td>
 							<td>A valid URL string&#160;[[!url]].</td>
```

Comment:
Maybe, to emphasize, we could say "Absolute URL of the profile"

Reply:
For the value, I can just change "valid URL string" to "absolute-URL-with-fragment string".